packages/cadl-ranch-specs/http/models/input-basic/main.cadl

```diff
@@ -10,7 +10,7 @@ using Azure.Core.Foundations;
 @server("http://localhost:3000", "TestServer endpoint")
 @doc("Illustrates an input-only model with required reference and value type properties.")
 @route("/input-basic")
-namespace Cadl.TestServer.InputBasic;
+namespace Models.InputBasic;
```

Comment:
I think we should go with just `Models.Input`. I don't htink `basic` adds anything

Also I'm going to be restructuring the model tests so there aren't a bunch of different cadl folders for little scenarios. Here's the plan: https://github.com/Azure/cadl-ranch/pull/55#discussion_r956069939, it will mainly be a re-org of the existing tests